bucket/free-shooter.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": "2.0.7",
+    "description": "Taking screenshots without bloatware features",
+    "homepage": "https://www.henrypp.org/product/freeshooter",
+    "license": "GPL-3.0-only",
+    "url": "https://github.com/henrypp/freeshooter/releases/download/v.2.0.7/freeshooter-2.0.7-bin.zip",
+    "hash": "9baaeffd561655696a6f925b6a2e166bb0e06897cec2b84d3e36cbbdc6a9efff",
+    "architecture": {
+        "64bit": {
+            "extract_dir": "freeshooter\\64"
+        },
+        "32bit": {
+            "extract_dir": "freeshooter\\32"
+        }
+    },
+    "pre_install": "if (!(Test-Path \"$persist_dir\\freeshooter.ini\")) { New-Item \"$dir\\freeshooter.ini\" | Out-Null }",
+    "bin": "freeshooter.exe",
+    "shortcuts": [
+        [
+            "freeshooter.exe",
+            "Free Shooter"
+        ]
+    ],
+    "persist": [
+        "freeshooter.ini",
+        "freeshooter.lng"
```

Comment:
Is persisting language file good idea/needed?

Application prompt for update of language pack on start.